client/css/style.css

```diff
@@ -293,6 +294,16 @@ kbd {
 	content: "\f005"; /* http://fontawesome.io/icon/star/ */
 }
 
+#chat .toggle-button {
+	/* These 2 directives are directly from .fa-fw */
+	width: 1.28571429em;
```

Comment:
I don't think we need such a precise width, the button can be wider to hit it easily.